src/config.js

```diff
@@ -89,7 +89,9 @@ export function getConfig() {
   if (process.env.BSK_SUBDOMAIN_OWNER_KEY) {
     config.ownerKey = process.env.BSK_SUBDOMAIN_OWNER_KEY
   }
-
+  if (process.env.ADMIN_PASSWORD) {
+    config.ownerKey = process.env.ADMIN_PASSWORD || configDevelopDefaults.ADMIN_PASSWORD
```

Comment:
`configDevelopDefaults.ADMIN_PASSWORD` is not defined

```suggestion
    config.ownerKey = process.env.ADMIN_PASSWORD || ADMIN_PASSWORD
```

Reply:
wouldn't that be from here? https://github.com/blockstack/subdomain-registrar/blob/fix/adminpass-var/src/developmode.js#L9

Reply:
No, `configDevelopDefaults` is a variable defined up above in this same file: https://github.com/blockstack/subdomain-registrar/blob/0b091b1f521af8f540960a4f74b915bfaa71c88b/src/config.js#L5